onward/app/views/fragments/rightMostPopularGeoGarnett.scala.html

```diff
@@ -8,39 +8,43 @@
 @import views.html.fragments
 @import model.pressed._
 
+@template1(trail: PressedContent, row: RowInfo) = {
+    <li class="right-most-popular-item" data-link-name="trail | @row.rowNum">
+        <a class="right-most-popular-item__url media u-cf" href="@LinkTo{@trail.header.url}">
+            @trail.trailPicture.map{ imageMedia =>
+                <div class="right-most-popular-item__img media__img">
+                    <img class="responsiveimg" src="@ImgSrc.getFallbackUrl(imageMedia)" alt="" />
+                </div>
+            }
+            <div class="media__body right-most-popular__content">
+                <h4 class="right-most-popular-item__headline">
+                    @if(trail.properties.isLiveBlog) {
+                        <span class="right-most-popular__kicker">Live</span>
+                    }
+                    @trail.header.headline
+                </h4>
+                @if(trail.properties.showByline) {
+                    <span class="
+                                    right-most-popular__byline
+                                    right-most-popular__byline--opinion
```

Comment:
the formatting of this `class` value is strange? I realise it hasn't changed though.

Reply:
Yep. I agree :)